diff --git a/MonoTouch.SQLite/SQLite.cs b/MonoTouch.SQLite/SQLite.cs
index df8daa7..4ca1a35 100644
--- a/MonoTouch.SQLite/SQLite.cs
+++ b/MonoTouch.SQLite/SQLite.cs
@@ -44,7 +44,7 @@ using Sqlite3DatabaseHandle = System.IntPtr;
 using Sqlite3Statement = System.IntPtr;
 #endif
 
-namespace SQLite
+namespace MonoTouch.SQLite
 {
 	public class SQLiteException : System.Exception
 	{
@@ -387,7 +387,7 @@ namespace SQLite
 		/// <returns>
 		/// A <see cref="SQLiteCommand"/>
 		/// </returns>
-		public SQLiteCommand CreateCommand (string cmdText, params object[] ps)
+		public SQLiteCommand CreateCommand (string cmdText, object[] ps)
 		{
 			if (!_open) {
 				throw SQLiteException.New (SQLite3.Result.Error, "Cannot create commands from unopened database");
@@ -983,7 +983,7 @@ namespace SQLite
 
 		public abstract class Column
 		{
-			public string Name { get; protected set; }
+			public string Name { get; internal set; }
 
 			public Type ColumnType { get; protected set; }
 
